Makefile

```diff
@@ -6,7 +6,7 @@ BUILD_NUMBER ?= 1
 
 CONFLUENT_MAJOR_VERSION ?= 4
 CONFLUENT_MINOR_VERSION ?= 0
-CONFLUENT_PATCH_VERSION ?= 1
+CONFLUENT_PATCH_VERSION ?= 3
```

Comment:
Sorry for the extra legwork, forgot this should actually be a seperate push to keep the merge cleans.